lib/generators/react_on_rails/install_generator.rb

```diff
@@ -20,19 +7,25 @@ class InstallGenerator < Rails::Generators::Base
       class_option :redux,
                    type: :boolean,
                    default: false,
-                   desc: "Setup Redux files",
+                   desc: "Install Redux gems and Redux version of Hello World Example",
                    aliases: "-R"
       # --server-rendering
       class_option :server_rendering,
                    type: :boolean,
                    default: false,
-                   desc: "Configure for server-side rendering of webpack JavaScript",
+                   desc: "Add necessary files and configurations for server-side rendering",
                    aliases: "-S"
-      # --linters
-      class_option :linters,
+      # --skip-js-linters
+      class_option :skip_js_linters,
```

Comment:
We have now discussed in Slack and decided to use the Rails `new` generator API as our example to emulate since it will be familiar for users.

https://github.com/rails/rails/blob/master/railties/lib/rails/generators/app_base.rb